Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19-alpine as build
+FROM golang:1.20.2-alpine as build
```

Comment:
Same here, if 1.20.3 comes out 1.20 will point to that, 1.20.2 won't.

Minor releases are always only bugfixes, usually security relevant and shouldn't break any existing software.